app/src/main/java/de/westnordost/streetcomplete/quests/foot/AddProhibitedForPedestrians.kt

```diff
@@ -34,6 +34,8 @@ class AddProhibitedForPedestrians : OsmFilterQuestType<ProhibitedForPedestriansA
     override val icon = R.drawable.ic_quest_no_pedestrians
     override val isSplitWayEnabled = true
 
+    override val enabledInCountries = AllCountriesExcept("SE")
```

Comment:
you'll need to import `de.westnordost.streetcomplete.data.quest.AllCountriesExcept` for this to work